core/lib/globals_shared.h

```diff
@@ -1912,8 +1918,9 @@ typedef union _dr_simd_t {
 #        define PRE_XMM_PADDING \
             24 /**< Bytes of padding before xmm/ymm dr_mcontext_t slots */
 #    endif
-#    define MCXT_NUM_OPMASK_SLOTS                                   \
-        8 /**< Number of 16-64-bit OpMask Kn slots in dr_mcontext_t \
+#    define MCXT_NUM_OPMASK_SLOTS                                    \
+        8 /**< Number of 16-64-bit OpMask Kn slots in dr_mcontext_t, \
```

Comment:
It seems weird to set this to 8 when the real number is 0 as there's nothing in dr_mcontext_t, but I guess this is being staged and the dr_mcontext_t addition will happen before any release or other point where everything must be consistent.

Reply:
I will add this to mcontext very soon.